commcare_connect/opportunity/models.py

```diff
@@ -777,10 +782,14 @@ class Meta:
 class ExchangeRate(models.Model):
     currency_code = models.CharField(max_length=3)
     rate = models.DecimalField(max_digits=10, decimal_places=6)
-    rate_date = models.DateField()
+    rate_date = models.DateField(db_index=True)
     fetched_at = models.DateTimeField(auto_now_add=True)
 
     class Meta:
         constraints = [
             models.UniqueConstraint(fields=["currency_code", "rate_date"], name="unique_currency_code_date")
         ]
+
+    @classmethod
+    def latest_exchange_rate(cls, currency_code, date):
```

Comment:
We are specifying a date arg and then prefixing the method with `latest` which seems incorrect.